docs/detections/rules-ui-create.asciidoc

```diff
@@ -450,9 +450,10 @@ the rule. For example, links to background information.
 .. *False positive examples* (optional): List of common scenarios that may produce
 false-positive alerts.
 .. *MITRE ATT&CK^TM^ threats* (optional): Add relevant https://attack.mitre.org/[MITRE] framework tactics, techniques, and subtechniques.
-.. *Custom highlighted fields* (optional): Specify highlighted fields for personalized alert investigation flows. Fields with values are added to the <<investigation-section,Highlighted fields>> section within the alert details flyout. Fields without values aren't added. After you create the rule, you can find all custom highlighted fields in the About section of the rule details page.
+.. *Custom highlighted fields* (optional): Specify one or more highlighted fields for personalized alert investigation flows. You can choose any fields that are available in the <<update-sec-indices,default {elastic-sec} indices>>. There is no limit to the number of fields that you can add. 
```

Comment:
This is incorrect. Here we will use fields from indices selected for the rule and only in case of bulk action we will use default indices. So, to summarize:
- Edit rule settings > About section: we use indices selected in the rule settings
- Bulk action: we use default indices from advanced security solution settings

That is why we also should not put `There is no limit to the number of fields that you can add.` here.

Reply:
Gotcha - makes sense and thanks for clarifying this. Here's the corrected version -

```suggestion
.. *Custom highlighted fields* (optional): Specify one or more highlighted fields for personalized alert investigation flows. You can choose any fields that are available in the indices you selected for the rule's data source.  
```